Tests/SwiftRedisTests/TestTransactionsPart5.swift

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import SwiftRedis
+
+import Foundation
+import XCTest
+
+//Tests the Set operations
+public class TestTransactionsPart5: XCTestCase {
+    static var allTests: [(String, (TestTransactionsPart5) -> () throws -> Void)] {
+        return [
+            ("test_keyUnion", test_keyUnion),
+            ("test_Move", test_Move),
+            ("test_Diffstorespop", test_Diffstorespop)
```

Comment:
Where are the rest of the tests? You have only tested some of the APIs. I realize that these tests each test multiple APIs but not all have been tested.